admin/views/tabs/metas/paper-content/author-archive-settings.php

```diff
@@ -81,5 +81,14 @@
 );
 
 $editor->render();
+
+/**
+ * Allow adding custom fields to the admin meta page - Author Archives tab.
+ *
+ * @since 16.2
+ *
+ * @api WPSEO_Admin_Pages $yform The WPSEO_Admin_Pages object
```

Comment:
`@api` is a legacy annotation, we should use `@param`
See https://yoast.slack.com/archives/CQ2G9ASK1/p1598262676005400